Makefile

```diff
@@ -358,6 +358,7 @@ genpdata:
 check-contrib:
 	@echo Setting contrib at $(CONTRIB_PATH) to use this core checkout
 	make -C $(CONTRIB_PATH) for-all CMD="go mod edit -replace go.opentelemetry.io/collector=$(CURDIR)"
+	make -C $(CONTRIB_PATH) gotidy
```

Comment:
Can be extracted in a separate PR :)

Reply:
https://github.com/open-telemetry/opentelemetry-collector/pull/3392